### ISSUE TITLE
[Lec8] 넘파이는 직접 Gradient 계산해야되는데 텐서플로우나 파이토치는 자동으로 해준다?

### ISSUE THREAD

**Issue by positive235** [ai-robotics-kr/CS231n_study#26]
## 넘파이는 직접 그래디언트 계산해야되는데 텐서플로우나 파이토치는 자동으로 해준다 ?

> 네 맞습니다 미분식을 적용하지 않아도 자동으로 계산해준다고 합니다. 텐서플로우 2.0에서는 gradients가 [[GradientTape]](https://www.tensorflow.org/tutorials/eager/automatic_differentiation?hl=ko)으로 쓰입니다. 

- [[Autograd 미분 자동화] 링크](http://taewan.kim/trans/pytorch/tutorial/blits/02_autograd/)
